Documentation/etcd-mixin/test.yaml

```diff
@@ -71,18 +71,16 @@ tests:
       - series: 'up{job="etcd",instance="10.10.10.1"}'
         values: '1 1 1 1 1 1 1 1 0 0 0 0 0 0 0 0'
```

Comment:
Do we want to pad these out with additional entries so they're all the same length?

Reply:
Go figure, fixing the padding fixed the (1) / (2) regression...